src/components.jl

```diff
@@ -648,11 +648,11 @@ function _relative_position(xmin, xmax, pos::Length{:pct}, scale::Symbol)
 
     # !TODO Add more scales in the future (asinh, sqrt) ?
     if scale === :log || scale === :ln
-        exp(log(xmin) + pos.value * (log(xmax) - log(xmin)))
+        exp(log(xmin) + pos.value * (log(xmax / xmin)))
```

Comment:
Parentheses around `log`s can be removed, and for `log10` and `log2` also..

Reply:
Just did it